src/main/java/seedu/address/model/person/Interaction.java

```diff
@@ -49,6 +54,20 @@ public Interaction(String interactionNote) {
     public Interaction(String interactionNote, Outcome outcome) {
         this.interactionNote = interactionNote;
         this.outcome = outcome;
+        this.date = new Date();
+    }
+
+    /**
+     * Constructs a {@code Interaction}.
+     *
+     * @param interactionNote A valid Interaction note.
+     * @param outcome A valid LeadStatus.
+     * @param date A valid date.
```

Comment:
This will only be necessary if the date needs to be explicitly specified right, which in our case it seems like as of now is not needed

Reply:
Yeap! But we can just leave it in?